scripts/switcher_gen.py

```diff
@@ -0,0 +1,89 @@
+import os
+import git
+import json
+import re
+
+
+# Filter function for building latest versions of each major tag.
+# For example, out of ["v0.8.0", "v0.8.1", "v0.9.2", "v0.9.3"]
+# those would be ["v0.8.1", "v0.9.3"]
+def latest_tags_filter(tag_list: list, start_version: str = "v0.8.0") -> list:
```

Comment:
Move the comments below the function as a docstring.

Reply:
2855d34